test/ICSharpCode.SharpZipLib.Tests/Zip/FastZipHandling.cs

```diff
@@ -187,8 +193,23 @@ public void Encryption()
 					ZipEntry entry = zf[0];
 					Assert.AreEqual(tempName1, entry.Name);
 					Assert.AreEqual(1, entry.Size);
-					Assert.IsTrue(zf.TestArchive(true));
+					// Assert.IsTrue(zf.TestArchive(true));
```

Comment:
```suggestion
					Assert.IsTrue(zf.TestArchive(true));
```
Could you leave this enabled? It would be a nice indicator for when `TestArchive` is actually fixed (although, we should probably have more tests for that).

Reply:
sure (there might be another PR that has it disabled too, will have to remind myself)